### ISSUE TITLE
[Housekeeping] Cookbook a11y issue

### ISSUE THREAD

**Issue by jakobe** [kirbydesign/designsystem#3537]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the housekeeping request
Running a full page scan of the Kirby Cookbook homepage using Deque's axe DevTools results in the following a11y issue:

![image](https://github.com/user-attachments/assets/43529651-b22a-48da-a64b-10287af75f3c)
 

Should be fixed according to the recommendation(s) in the test result ☝🏻 

## Have you considered any alternatives?

No.

## Are there any additional context?

No.
